.github/workflows/publish-ci.yml

```diff
@@ -13,6 +13,7 @@ concurrency:
 jobs:
   publish:
     runs-on: ubuntu-22.04
+    if: ${{ github.repository_owner == 'deephaven' }}
```

Comment:
Added because I dont think we want to attempt any deploys for tags that exist on forks

Reply:
I'm not opposed, but it would have to be commented out to test _Publish to Maven Local_ and _Upload Artifacts_ steps.  Maybe it would be clearer with a global variable like _allowPublish_ applied to just the steps that need the protection?